chain/src/tests_utils.rs

```diff
@@ -249,7 +243,8 @@ impl TestChainCluster {
                 .unwrap()
                 .take_events_stream()
                 .unwrap();
-            self.runtime.spawn(async move {
+
+            tokio::spawn(async move {
```

Comment:
I wrapped `tokio::spawn` in `exocore_core::futures::spawn_future` to allow switching executor (needed for wasm). Doesn't change much for tests, but just for consistency.